### DISCUSSION TITLE
Attribute default & return_obj

### DISCUSSION THREAD

**Discussion by davewiththenicehat** [evennia/evennia#2394] [Evennia Engine Development]
If `default` and `return_obj` kwargs both have values should `return_obj` return None or the instance of `default`?
Example:
`self.db.get(db_key, default=db_key_def_val, return_obj=True)`
Returns None if there is no data base field. Unless `return_list` is `True`. It looks like it would return default in a list.
Not a terribly big issue as it requires one line to turn the return into default.
I'm not certain if people want the None instance over their default here.

I am more than happy to make the change and commit it.
Figured I would ask before making a branch and creating a commit.

**Comment by Griatch** [comment]
It would probably make more sense to actually return the `default` if it is given, also if `return_obj=True`. Supposedly the caller knows what kwargs they put in and makes sure the default is what they want it to be :grin:  So yeah, this could be a good fix.
